examples/wearable/UIComponents/contents/presentation-views/list/main/3-line.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html>
```

Comment:
this should be in sub-text folder I think

Reply:
I thought about adding examples to sub-text manu only  from Main + SubText as per guide but 2, 3, 4 sub text lines look similar and we can change this I think.

Reply:
Done. In addition changed sub-text/sub-text.html to sub-text/1-line.html.

Reply:
makes sense